test/sql/CMakeLists.txt

```diff
@@ -54,11 +54,16 @@ set(TEST_TEMPLATES
 # Regression tests that vary with PostgreSQL version. Generated test
 # files are put in the original source directory since all tests must
 # be in the same directory.
+if (${PG_VERSION_MAJOR} EQUAL "10")
```

Comment:
I don't get this. why does ${PG_VERSION_MINOR} depend on 10 or not?

Reply:
Because we support all PG 10 minor versions, but only 9.6 for PG9. Presumably we could even extend our inclusion to 9.5 if it was demanded enough so I wanted to keep it around. Basically the switch for PG to drop one of the numbers changes things and I want it clear that this is 9.6 not all of PG9